### DISCUSSION TITLE
How to create innerHTML that fills 100% of the region

### DISCUSSION THREAD

**Discussion by Serek16** [katspaugh/wavesurfer.js#3492] [Q&A] [answered]
Hi, I'd like to add to the region.content.innerHTML div that can stay at 100% of region's space. So it would always change size and adjust depending on zoom level. 
My goal is to create an empty div with title attribute so it would give mouseover text to my regions. I'd like to hover anywhere on the region to message to pop up

![obraz](https://github.com/katspaugh/wavesurfer.js/assets/50013854/dfb92880-2345-466d-bda9-a49ec71749a3)

**Comment by katspaugh** [comment, marked as answer]
```js
const div = document.createElement('div')
div.style.width = '100%'
div.innerText = 'empty div'
wavesurferRegions.addRegion({
  start: 1,
  end: 3,
  content: div
})
```

**Reply by Serek16** [reply]
I see. I tried different approach and it didn't work but now it seems to be easier than I thought.
